### DISCUSSION TITLE
FOUC with TailWind on Firefox

### DISCUSSION THREAD

**Discussion by younggraham-t** [vercel/next.js#70673] [Help]
### Link to the code that reproduces this issue

https://github.com/younggraham-t/nextjs-chess/tree/dev

### To Reproduce

1.  start the application 
2. refresh page in firefox

### Current vs. Expected behavior

![simplescreenrecorder-2024-09-26_13 37 02-ezgif com-optimize](https://github.com/user-attachments/assets/b877b4f2-9b68-40e9-90d2-fef63d683123)
when the page is loaded there is a flash of unstyled content (FOUC) in firefox but not in chrome

### Provide environment information

```bash
Operating System:
  Platform: linux
  Arch: x64
  Version: #45-Ubuntu SMP PREEMPT_DYNAMIC Fri Aug 30 12:02:04 UTC 2024
  Available memory (MB): 3819
  Available CPU cores: 4
Binaries:
  Node: 22.7.0
  npm: 10.8.2
  Yarn: N/A
  pnpm: 9.8.0
Relevant Packages:
  next: 15.0.0-canary.171 // Latest available version is detected (15.0.0-canary.171).
  eslint-config-next: 14.2.8
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Which stage(s) are affected? (Select all that apply)

next dev (local)

### Additional context

The issue only seems to happen on firefox. I have tested it on chrome and it was fine, and I have not tested any other browsers. Also I'm using tailwind css if that matters.

**Comment by samcx** [comment]
@younggraham-t I was able to replicate this on Chrome. Did you try to throttle your speed (dev tools) on Chrome?
